src/crypto.rs

```diff
@@ -54,6 +54,7 @@ impl Crypto {
         }
 
         let mut ssl_acceptor_builder = SslAcceptor::mozilla_intermediate(SslMethod::dtls())?;
+        ssl_acceptor_builder.set_tlsext_use_srtp("SRTP_AES128_CM_SHA1_80:SRTP_AES128_CM_SHA1_32:SRTP_AEAD_AES_128_GCM:SRTP_AEAD_AES_256_GCM").unwrap();
```

Comment:
I don't think sha1 should be supported. All major browsers have removed support for it by now due to hash collisions having been found. Also maybe but aes 256 before aes 128?

Reply:
Makes sense, done.
Btw, I didn't know that the order means something. Do peers pick the first common suitable protocol during a handshake?

Reply:
I believe so.